.ebextensions/deploy.config

```diff
@@ -7,3 +7,11 @@ packages:
 commands:
     01-upgrade-pip:
         command: "/opt/python/run/venv/bin/pip  install --upgrade pip"
+
+files:
+    "/etc/logrotate.elasticbeanstalk.hourly/logrotate.elasticbeanstalk.httpd.conf":
+        mode: "000644"
```

Comment:
These were set to the same as was already on the instance. I've removed as they are not needed.